assets/javascripts/wizard/models/custom.js.es6

```diff
@@ -110,12 +94,45 @@ export function findCustomWizard(wizardId, params = {}) {
       Discourse.Site.currentProp("categoriesById", categoriesById);
       Discourse.Site.currentProp(
         "uncategorized_category_id",
-        wizard.uncategorized_category_id
+        wizardJson.uncategorized_category_id
       );
     }
 
-    return CustomWizard.create(wizard);
+    return CustomWizard.create(wizardJson);
+  },
+});
+
+export function findCustomWizard(wizardId, params = {}) {
```

Comment:
findCustomWizard -> fetchCustomWizard
The naming convention should suggest that the function is bring the data from the server. Thoughts?

Reply:
@fzngagan Maybe. This naming convention is bot pre-existing in this repo, and [copied from the core wizard](https://github.com/discourse/discourse/blob/master/app/assets/javascripts/wizard/models/wizard.js#L87) though, so I don't think we should change that in this PR.